### DISCUSSION TITLE
Assistance Installing on Manjaro

### DISCUSSION THREAD

**Discussion by dragonshardz** [abraunegg/onedrive#2779] [Q&A] [answered]
Hi, I'm trying to install this and when I run `pamac build onedrive-abraunegg`, I receive this error: `Failed to read AUR data from /var/tmp/pamac/dbs/sync/packages-meta-ext-v1.json.gz : JSON data must be UTF-8 encoded`.

If I try to install via `yay`, instead, I get an error stating ` -> Failed to install the following packages. Manual intervention is required: onedrive-abraunegg - exit status 15`

I'm not sure why this is happening, but some help resolving the issue would be appreciated

**Comment by dragonshardz** [comment, marked as answer]
I checked the dependencies and installed them, then the instructions worked. How about that.

**Comment by abraunegg** [comment]
Glad you have got this resolved, and, as per the [installation documentation](https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-3/docs/install.md) you need to install certain prerequisites to ensure the package is built and installed on your platform:

![image](https://github.com/user-attachments/assets/29da032e-840a-4e11-ae2a-700a22436d0e)
